modules/openapi-generator/src/main/resources/csharp-netcore/ApiClient.mustache

```diff
@@ -19,18 +19,28 @@ using System.Web;
 {{/netStandard}}
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
+using ErrorEventArgs = Newtonsoft.Json.Serialization.ErrorEventArgs;
+{{#useRestSharp}}
```

Comment:
I'll create a separate ApiClient.mustache just for `httpclient` library similar to how we support multiple HTTP libraries in other client generators as too many `{{#useRestSharp}} ... {{/useRestSharp}}` etc make the template hard to maintain.